include/clang/Interpreter/InterOp.h

```diff
@@ -164,6 +164,8 @@ namespace InterOp {
   CallFuncWrapper_t GetFunctionCallWrapper(TInterp_t interp,
                                            TCppFunction_t func);
 
+  std::string GetFunctionArgDefault(TCppFunction_t func, TCppIndex_t param_index);
```

Comment:
```suggestion
  /// Returns the default argument value as string.
  std::string GetFunctionDefaultArgValueAsString(TCppFunction_t func, TCppIndex_t param_index);
```

Reply:
I have made this changes as mentioned by you sir
I hope this is correct now